src/Features/CSharp/Portable/FileHeaders/CSharpFileHeaderHelper.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Microsoft.CodeAnalysis.FileHeaders;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.CSharp.FileHeaders
+{
+    /// <summary>
+    /// Helper class used for working with file headers.
+    /// </summary>
+    internal sealed class CSharpFileHeaderHelper : AbstractFileHeaderHelper
+    {
+        public static CSharpFileHeaderHelper Instance { get; } = new CSharpFileHeaderHelper();
+
+        private CSharpFileHeaderHelper()
+        {
+        }
+
+        internal override int SingleLineCommentTriviaKind => (int)SyntaxKind.SingleLineCommentTrivia;
+        internal override int MultiLineCommentTriviaKind => (int)SyntaxKind.MultiLineCommentTrivia;
+        internal override int WhitespaceTriviaKind => (int)SyntaxKind.WhitespaceTrivia;
+        internal override int EndOfLineTriviaKind => (int)SyntaxKind.EndOfLineTrivia;
+        internal override string CommentPrefix => "//";
```

Comment:
internal?  seems odd.  we *rarely* have explicit internal members in IDE.

Reply:
changed